### ISSUE TITLE
Sign a Windows binary - any plan?

### ISSUE THREAD

**Issue by julianxhokaxhiu** [actions/attest-build-provenance#64]
Hello,

I got very excited but your latest announcement at https://github.blog/2024-05-02-introducing-artifact-attestations-now-in-public-beta/ and I was wondering, since now you've established your own CA, is there any plan to sign Windows binaries to at least allow FOSS projects bypass the Windows consent screen and false-positive warnings in Windows Defender ( and most anti viruses anyway )?
Basically an alternative to https://signpath.org/about/ but made in Github.

I see this action fitting this purpose perfectly, but as I see nothing mentioned around Windows I thought it would be worth to ask anyway.

Thank you in advance and best regards,
Julian

**Comment by phillmv** [comment]
Thanks for reaching out!

Unfortunately, that's beyond the scope of this project 😞.
